hnn_core/cell.py

```diff
@@ -202,6 +203,14 @@ def __init__(self, L, diam, Ra, cm, end_pts=None):
         self.mechs = dict()
         self.syns = list()
 
+        # For distance functionality
```

Comment:
For this function as well as any relevant ones below, can you add a comment with a URL to the relevant C++ code in NEURON? Think it will be useful to have the origin of this documented

Reply:
Sure will add them